.github/wiki/Trivy-local-running.md

```diff
@@ -0,0 +1,12 @@
+This project is using Trivy as a vulnerability scanner to replace the role of `tfsec` with some extra benefits:
+1. Access to more languages and features in the same tool.
+2. Access to more integrations with tools and services through the rich ecosystem around Trivy.
+3. Commercially supported by Aqua as well as by a the passionate Trivy community. tfsec will continue to remain available for the time being, although our engineering attention will be directed at Trivy going forward.
```

Comment:
> our engineering attention

it is misleading sentence because we are not developing tfsec or trivy

Reply:
It introduces the reason we migrated to `trivy` from `tfsec` and shows beneficial points. 
Removed the third point to keep things short in [b22f6f4](https://github.com/nimblehq/infrastructure-templates/pull/262/commits/b22f6f410338fc7dc12445e142ab928cf2345919)